packages/camera/android/src/main/java/io/flutter/plugins/camera/FilePathValidator.java

```diff
@@ -0,0 +1,48 @@
+package io.flutter.plugins.camera;
+
+import java.io.File;
+
+class FilePathValidator {
```

Comment:
Can you write a small javadoc explaining what the class does and that `validate()` must be called before calling `isValid()` or `getErrorMessage()`?